test/DynamoCoreTests/SerializationTests.cs

```diff
@@ -52,6 +64,14 @@ public void TearDown()
             ExecutionEvents.GraphPostExecution -= ExecutionEvents_GraphPostExecution;
         }
 
+        private void DeleteJsonFilesFromTemp(string[] files)
+        {
+            foreach (var file in files)
+            {
+                File.Delete(file);
```

Comment:
this should probably be made more tolerant, assert that the files actually exist before you try this and check if the deletion succeeds, you may not have permissions when this call is made.